### PR TITLE
enh: add comsumer get topic

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM

File: src/main/java/com/taosdata/jdbc/TSDBStatement.java [hunk 1]
```diff
@@ -93,11 +91,9 @@ public int executeUpdate(String sql) throws SQLException {
 
             try {
                 return f.get(this.queryTimeout, TimeUnit.SECONDS);
-            } catch (InterruptedException e) {
-                throw new SQLException("failed to execute sql: " + sql + ", cause: " + e.getMessage(), e);
-            } catch (ExecutionException e) {
+            } catch (InterruptedException | ExecutionException e) {
                 throw new SQLException("failed to execute sql: " + sql + ", cause: " + e.getMessage(), e);
-            } catch (TimeoutException e) {
+            }  catch (TimeoutException e) {
```
Comment (sangshuduo) [inline src/main/java/com/taosdata/jdbc/TSDBStatement.java:96]: extra space

Comment (huolibo) [reply src/main/java/com/taosdata/jdbc/TSDBStatement.java]: done

**Review by zyyang-taosdata — APPROVED** [review]
LGTM

**Pull request merged**